superset/assets/spec/javascripts/sqllab/ExploreResultsButton_spec.jsx

```diff
@@ -55,13 +55,11 @@ describe('ExploreResultsButton', () => {
   const mockColumns = {
     ds: {
       is_date: true,
-      is_dim: false,
```

Comment:
@mistercrunch removing `is_dim` everywhere as it doesn't appear to be used any longer.

Reply:
kill!

Reply:
I think it may map to that right-most checkbox, and define whether things show up in `groupby` controls
<img width="939" alt="Screen Shot 2019-12-12 at 11 01 42 PM" src="https://user-images.githubusercontent.com/487433/70776955-6ac42b00-1d33-11ea-85f0-c51af5d451df.png">

Reply:
Yup, it used to map to `BaseColumn.groupby`, but not anymore. Seems we can remove all refs to `is_dim`